resources/views/errors/layout.blade.php

```diff
@@ -15,10 +13,10 @@
                 margin: 0;
                 padding: 0;
                 width: 100%;
-                color: #B0BEC5;
+                color: #999999;
```

Comment:
Why changing the color here?

Reply:
The colour #B0BEC5 is only used in that one instance – so I replaced it with its nearest neighbour as part of the standardisation process.